build

```diff
@@ -109,20 +109,11 @@ Dir.chdir(File.dirname(__FILE__)) do
 
   add_license("pkg/sinon-ie#{version_string}.js", version)
 
-  FileUtils.cp("lib/sinon/util/fake_timers.js", "pkg/sinon-timers#{version_string}.js")
-  add_license("pkg/sinon-timers#{version_string}.js", version)
-  FileUtils.cp("lib/sinon/util/timers_ie.js", "pkg/sinon-timers-ie#{version_string}.js")
-  add_license("pkg/sinon-timers-ie#{version_string}.js", version)
-
   merger = Juicer::Merger::JavaScriptMerger.new
   merger << "lib/sinon/util/fake_server_with_clock.js"
   merger.save("pkg/sinon-server#{version_string}.js")
   add_license("pkg/sinon-server#{version_string}.js", version)
   FileUtils.cp(output, 'pkg/sinon.js')
-  FileUtils.cp("pkg/sinon-ie#{version_string}.js", 'pkg/sinon-ie.js')
```

Comment:
We probably want to keep making `sinon-ie.js` and `sinon-server.js` for a little while longer.